libraries/tests/rtos/mbed/mutex/main.cpp

```diff
@@ -16,8 +16,7 @@
     #define STACK_SIZE DEFAULT_STACK_SIZE
 #endif
 
-void print_char(char c = '*')
-{
```

Comment:
I do not understand, which style here is the most popular one?
I'm alligning all tests to the same style which is something like this:
```C++
void function(int param1, char param2 = '*') {
    /* FUNCTION BODY */
}
```